pages/docs/user-guide/compile-kernel.mdx

```diff
@@ -0,0 +1,49 @@
+# Compiling Linux kernel
+
+Lambdo uses a Linux kernel compiled from source. This is a guide to compile the
+kernel.
+
+## Install dependencies
+
+#### Debian/Ubuntu
+
+```bash copy filename="bash"
+sudo apt update
+sudo apt install git build-essential bc flex bison curl tar
+```
+
+#### Fedora/RHEL/CentOS
+
+```bash copy filename="bash"
+sudo dnf install git gcc make xz bc flex bison diffutils curl tar
+```
+
+#### Arch Linux
+
+```bash copy filename="bash"
+sudo pacman -Syu git gcc make xz bc flex bison diffutils curl tar
+```
+
+## Build the kernel
+
+Download and extract the kernel source:
+
+```bash copy filename="bash"
+curl -L https://cdn.kernel.org/pub/linux/kernel/v6.x/linux-6.5.9.tar.xz --output linux-6.5.9.tar.xz
+tar xf linux-6.5.9.tar.xz --directory linux
```

Comment:
Here I got an error the directory linux must created before:

```
curl -L https://cdn.kernel.org/pub/linux/kernel/v6.x/linux-6.5.9.tar.xz --output linux-6.5.9.tar.xz
tar xf linux-6.5.9.tar.xz --directory linux
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  132M  100  132M    0     0  25.5M      0  0:00:05  0:00:05 --:--:-- 27.2M
tar: linux: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
```

You should add a command to create the directory to be sure it will work in any case

```suggestion
mkdir linux
tar xf linux-6.5.9.tar.xz --directory linux
```